.github/CODEOWNERS

```diff
@@ -841,3 +841,4 @@
 /eng/emitter-package-lock.json                                       @mccoyp @catalinaperalta @kristapratico @iscai-msft
 
 /pylintrc                                                            @l0lawrence
+/sdk/**/ci.yml                                                       @msyyc @lmazuel
```

Comment:
I think this might be a little too broad. What is your goal?

cc @benbp

Reply:
Just read the description. I think I'm OK with this change as generally these ci.yml files are boilerplate. Do keep in mind that you will get added for all new libraries not just mgmt.

Reply:
Yes I'm fine with this, but if the noise of new ci.yml files for data plane gets to be too much then we may have to think of another solution.